### ISSUE TITLE
batch size for training should be read from config.yaml  (imagaug loader)

### ISSUE THREAD

**Issue by MMathisLab** [DeepLabCut/DeepLabCut#508]
**Is your feature request related to a problem? Please describe.**
There is no straightforward way to set `batch_size`>1 during training with `imgaug` loader if batch scripting runs. Currently, one needs to wait until the `pose_cfg.yaml` file is created during `create_training_set` then hand edit the file to include `batch_size = 8` (for example). 

**Describe the solution you'd like**
This should automatically read the `batchsize` variable from config.yaml during training set creation.

**Comment by AlexEMG** [comment]
I think this is not practical, as for inference and training typically require very different batch_sizes. From docs: 

DATA AUGMENTATION: At this stage you can also decide what type of augmentation to use. The default loaders work well for most all tasks (as shown on www.deeplabcut.org), but there are many options, more data augmentation, intermediate supervision, etc. Please look at the pose_cfg.yaml file for a full list of parameters you might want to change before running this step. There are several data loaders that can be used. For example, you can use the default loader (introduced and described in the Nature Protocols paper), TensorPack for data augmentation (currently this is easiest on Linux only), or imgaug. You can set this by passing:deeplabcut.create_training_dataset(config_path, augmenter_type='imgaug')

The differences of the loaders are as follows:

- default: our standard DLC 2.0 introduced in Nature Protocols variant (scaling, auto-crop augmentation)
- imgaug: a lot of augmentation possibilities, efficient code for target map creation & batchsizes >1 supported. [will prob. become default soon]. You can set the parameters such as the batch_size in the pose_cfg.yaml file for the model you are training.
- tensorpack: a lot of augmentation possibilities, multi CPU support for fast processing, target maps are created less efficiently than in imgaug, does not allow batchsize>1
- deterministic: only useful for testing, freezes numpy seed otherwise like default

Alternatively, you can set the loader (as well as other training parameters) in the pose_cfg.yaml file of the model that you want train. Note, to get details on the options, look at the default file: pose_cfg.yaml.
